lib/spack/spack/operating_systems/windows_os.py

```diff
@@ -16,8 +16,10 @@
 
 
 def windows_version():
-    """temporary workaround to return a Windows version as a Version object"""
-    return Version(platform.release())
+    """ Windows version as a Version object """
+    # include the build number as this provides important information
+    # for low lever packages and components like the SDK and WDK
```

Comment:
It might be worth expanding on this slightly like by talking about how the build number is component z of x.y.z.